src/server.h

```diff
@@ -581,6 +581,15 @@ typedef enum {
 #define PAUSE_ACTION_EVICT (1 << 3)
 #define PAUSE_ACTION_REPLICA (1 << 4) /* pause replica traffic */
 
+/* Sets log format */
+typedef enum { LOG_FORMAT_DEFAULT = 0, LOG_FORMAT_LOGFMT } log_format_type;
```

Comment:
Can we come up with another name for "default"? The name default implies that it will always be the default, which isn't necessarily true forever.

"compact", " traditional"...?

Reply:
The decision on this was to use "legacy".